src/imio/esign/acroform.py

```
@@ -0,0 +1,192 @@
+PDFTOTEXT = "pdftotext"
+
+# content type -> path, inside the zipped document, of the xml part holding the text
+ZIPPED_XML_PARTS = {
+    "application/vnd.oasis.opendocument.text": "content.xml",
+    "application/vnd.openxmlformats-officedocument.wordprocessingml.document": "word/document.xml",
+}
+
+
+def _pdf_text(data):
+    """Return the text of a pdf, read by the poppler pdftotext binary."""
+    process = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)